libs/wingsdk/src/shared/misc.ts

```diff
@@ -27,8 +27,10 @@ export async function runCommand(cmd: string, args: string[]): Promise<any> {
   const raw = await new Promise((resolve, reject) => {
     execFile(cmd, args, (error, stdout, stderr) => {
       if (error) {
-        console.error(stderr);
-        reject(error);
+        if (!(cmd === "docker" && args.includes("inspect"))) {
+          console.error(stderr);
+          reject(error);
+        }
```

Comment:
Because this is a general utility function, we probably shouldn't hardcode specific cases like this.
The root issue here is that this function has a side-effect (`console.error`). It's probably be better to remove that and instead return a more useful result. Then it can be the consumers responsibility to log or not.

This whole function could just be a simple wrapper around `promisify(execFile)` ([doc](https://nodejs.org/api/util.html#utilpromisifyoriginal)). The promise version of execFile has everything we could need in both the successful and rejected cases.